enable/gcbench/publish.py

```diff
@@ -201,8 +201,8 @@ def _format_timing(result):
 
 
 # HTML utils
-def _img(src):
-    return f'<img src="{src}" />'
+def _img(src, width, height):
+    return f'<img src="{src}" width="{width}" height="{height}"/>'
```

Comment:
HTML is doing all the work to make the 2x images appear normally.

Reply:
Maybe this / something similar should be a comment in the code?  It may be overkill though, so feel free to ignore

Reply:
Eh... I'm not that concerned.